### DISCUSSION TITLE
PostProcessor based execution stop

### DISCUSSION THREAD

**Discussion by srinath-chakravarthy** [idaholab/moose#16456] [Q&A General] [answered]
Hello all, 
Is there a way to stop program execution based on a postprocessor value or any other value ? 

Cheers
Srinath

**Comment by aeslaughter** [comment, marked as answer]
Yes, you need to use the Terminator!  It doesn't have documentation yet, but the tests should help you figure out how to use it.

https://mooseframework.inl.gov/source/userobject/Terminator.html

**Comment by srinath-chakravarthy** [comment]
Thanks andrew that works for me.

Cheers
Srinath
